pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -110,7 +110,8 @@ social:
             = hoc_s(:hoc_live_sign_up_title)
           %br/
           %div.banner-message
-            = hoc_s(:hoc_live_sign_up_message)
+            %p
+              = hoc_s(:hoc_live_sign_up_message, markdown: true, locals: {episodes_url: 'https://www.youtube.com/playlist?list=PLzdnOPI1iJNfSMYuR-d7C1wRhluXX7j7v'})
```

Comment:
if you're using `markdown: true`, the output gets wrapped in a paragraph by the markdown renderer (that's one of the major differences between `markdown: true` and `markdown: :inline` so with this change we actually _don't_ need the paragraph tag anymore :upside_down_face:

Reply:
haha TIL! thanks